examples/00-mapdl-examples/torsional_load.py

```diff
@@ -0,0 +1,201 @@
+"""
+.. _ref_tors_load:
+
+==============================================
+Torsional load on a bar using SURF154 elements
+==============================================
+
+This Ansys PyMAPDL script builds a bar and applies torque to it using
+SURF154 elements. This is a static analysis example.
+"""
+
+##############################################################################
+# Script initialization
+# ---------------------
+
+import os
+
+import numpy as np
+
+from ansys.mapdl.core import launch_mapdl
+
+# start Ansys in the current working directory with default jobname "file"
+mapdl = launch_mapdl(run_location=os.getcwd())
```

Comment:
Just to show we have that option. It should not change anything.

```suggestion
mapdl = launch_mapdl(run_location=os.getcwd(), version=23.1)
```